corehq/apps/users/tests/test_signals.py

```diff
@@ -1,24 +1,22 @@
 import uuid
+from unittest.mock import MagicMock, patch
 
 from django.test import SimpleTestCase
 
-from unittest.mock import MagicMock, patch
-
 from dimagi.utils.couch.undo import DELETED_SUFFIX
-# Also, you need to patch the path to the function in the file where the signal
-# handler uses it, not where it's actually defined.  That's quite a gotcha.
-from pillowtop.es_utils import initialize_index_and_mapping
 
-from corehq.apps.reports.analytics.esaccessors import get_user_stubs
+from corehq.apps.es.client import manager
 from corehq.apps.es.tests.utils import es_test
-from corehq.elastic import doc_exists_in_es, get_es_new
-from corehq.pillows.mappings.user_mapping import USER_INDEX_INFO
-from corehq.util.es.elasticsearch import ConnectionError
+from corehq.apps.es.users import user_adapter
+from corehq.apps.reports.analytics.esaccessors import get_user_stubs
 from corehq.util.es.testing import sync_users_to_es
-from corehq.util.test_utils import mock_out_couch, trap_extra_setup
+from corehq.util.test_utils import mock_out_couch
 
 from ..models import CommCareUser, WebUser
 
+# Also, you need to patch the path to the function in the file where the signal
+# handler uses it, not where it's actually defined.  That's quite a gotcha.
+
 # Note that you can't directly patch the signal handler, as that code has
 # already been called.  It's easier to patch something that the handler calls.
```

Comment:
It's funny that two iterations of `isort` have resulted in this comment getting split and re-joined in the reverse order from its [original form](https://github.com/dimagi/commcare-hq/commit/e5ea5c08bebd8df506e5e5f6dea991e19935e350#diff-6dd78305cdabcd8dc11acdb577a88227beb8012f43c27427920b7315afe1f42cR8-R12) :joy: 


```suggestion
# Note that you can't directly patch the signal handler, as that code has
# already been called.  It's easier to patch something that the handler calls.

# Also, you need to patch the path to the function in the file where the signal
# handler uses it, not where it's actually defined.  That's quite a gotcha.
```